public/docs/md/charts/en/group-by.md

```diff
@@ -59,4 +59,14 @@ For formulas *with* a group-by, Amplitude ranks the groups by the largest overal
 
 {{partial:admonition type='note'}}
 If group-by pruning occurs with multiple formula terms combined with operators, formulas may take longer to load because Amplitude runs additional queries to make sure that all formula terms are querying the same groups.
-{{/partial:admonition}}
\ No newline at end of file
+{{/partial:admonition}}
```

Comment:
Here are the feature flags https://app.amplitude.com/experiment/amplitude/318008/config/410052/configure https://app.amplitude.com/experiment/amplitude/187520/config/385224/configure. I have turned them on for you